### ISSUE TITLE
NullPointerException occur when commit migration job ~

### ISSUE THREAD

**Issue by MaricoHan** [apache/shardingsphere#21895]
## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?
5.2.1
### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-Proxy
### Expected behavior
**migrate single table; commit successfully without exception.**

### Actual behavior
If the source database doesn't generate incremental data, a null pointer exception will be reported ! (single table)

```java
java.lang.NullPointerException: Cannot invoke "org.apache.shardingsphere.data.pipeline.mysql.ingest.binlog.event.AbstractBinlogEvent.getFileName()" because "this.lastBinlogEvent" is null
```

**Comment by RaigorJiang** [comment]
Hi @MaricoHan 
Can you provide more logs and also what is output from `SHOW MIGRATION PROCESS CONFIGURATION`.

**Comment by ChenyuMa** [comment]
Same Question
When I migrate ds_1 to ds_0, it occured java.lang.NullPointerException
I don't know which yml config file is needed



![image](https://user-images.githubusercontent.com/28981436/200161243-e8122432-f527-427b-beb7-3168d3da5b9f.png)

![image](https://user-images.githubusercontent.com/28981436/200161168-f806dd3e-8d85-457f-b137-643d6d42270c.png)

**Comment by sandynz** [comment]
Hi @MaricoHan , thanks for your feedback. Is there full exception stack trace? It's important to analyze the problem, or could you supply the reproduce steps?

We've fixed a similar NPE in 5.2.1 by #21513.

**Comment by sandynz** [comment]
Hi @ChenyuMa , seems it migrate table from `ds_1` to `ds_0`, did you execute DistSQL `ADD MIGRATION SOURCE RESOURCE ds_1`?

Suppose you're running proxy 5.2.1. You could try to go through the [usage example]( https://shardingsphere.apache.org/document/5.2.1/cn/user-manual/shardingsphere-proxy/migration/usage/ ), make sure it works, and then you could modify the example for your real table.
